students/brgalloway/Lesson_6/test_mailroom_part_4.py

```diff
@@ -0,0 +1,71 @@
+from mailroom_part_4 import *
+import os
+
+a_dict = {
+        "Jeff Bezos": {
+            "donation_total": 877.33,
+            "times_donated": 1,
+            "average_donation": 877.33
+        },
+        "Paul Allen": {
+            "donation_total": 708.42,
+            "times_donated": 3,
+            "average_donation": 236.14
+        }
+    }
+
+'''
+ensure that list of names returned is a string
+'''
+def test_list_names():
+    names = list_names()
+    assert type(names) is str
```

Comment:
This is not a good test, don't check for the type, check for content of the string.